config/default/jenkins-infra.yaml

```diff
@@ -297,235 +297,95 @@ jenkins:
+                          }
+                          // Only Origin Pull Request
+                          gitHubPullRequestDiscovery {
+                            strategyId(1) // 1-Merging the pull request with the current target branch revision
+                          }
+                          gitHubExcludeArchivedRepositories()
+                          pullRequestLabelsBlackListFilterTrait {
+                            labels('on-hold ci-skip')
+                          }
+                          headWildcardFilterWithPR {
+                            includes('main master PR-*')
+                            excludes('')
+                            tagIncludes('*')
+                            tagExcludes('')
                           }
-                        }
-                        sourceRegexFilter {
-                          regex('docker-.*|^jenkins.io$')
-                        }
-                        gitHubBranchDiscovery {
-                          strategyId(1) // 1-Exclude branches that are also filed as PRs
-                        }
-                        // Only Origin Pull Request
-                        gitHubPullRequestDiscovery {
-                          strategyId(1) // 1-Merging the pull request with the current target branch revision
-                        }
-                        gitHubExcludeArchivedRepositories()
-                        pullRequestLabelsBlackListFilterTrait {
-                          labels('on-hold ci-skip')
-                        }
-                        headWildcardFilterWithPR {
-                          includes('main master PR-*')
-                          excludes('')
-                          tagIncludes('*')
-                          tagExcludes('')
                         }
                       }
                     }
-                  }
-
-                  orphanedItemStrategy {
-                    discardOldItems {
-                      daysToKeep(1) // Keep removed SCM heads/branch/PRs only for 1 day
-                    }
-                  }
-
-                  projectFactories {
-                    workflowMultiBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
-                    }
-                  }
-
-                  buildStrategies {
-                    buildAnyBranches {
-                      strategies {
-                        buildChangeRequests {
-                          ignoreTargetOnlyChanges(true)
-                          ignoreUntrustedChanges(true)
-                        }
-                        buildRegularBranches()
-                        buildTags {
-                          atLeastDays("0")
-                          atMostDays("7")
-                        }
+                    orphanedItemStrategy {
+                      discardOldItems {
+                        daysToKeep(1) // Keep removed SCM heads/branch/PRs only for 1 day
```

Comment:
One day seems a bit short to me, what do you think to put to 7 days?

Reply:
@olblak This setting is only about tracking the pipelines of the merged/closed PR or deleted branches. What would be the goal of keeping it for 7 days? (I would advocate for immediate deletion honestly 😅).
The build log persistence is a different setup, which is done on the pipelines themselves: it's up to the pipeline's writer to define this policy (e.g. no need to change the Jenkins conf for such a senstivie one).

Reply:
Discussed in 1:1 with @olblak , we keep the 1 day the comment was about the build retention

Reply:
Added comments in #1126  to help the future readers